backend/devices/usb/manager.go

```diff
@@ -280,7 +276,7 @@ func (manager *Manager) listen() {
 				continue
 			}
 			// Skip if we already have another device registered and we only support one device.
-			if manager.onlyOne && len(manager.devices) != 0 {
+			if len(manager.devices) != 0 {
 				continue
```

Comment:
I find this loop a bit useless. Would've done it the opposite way: run it only when `len(devices) == 0`.